modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/simulation/v0.1.0.json

```diff
@@ -0,0 +1,64 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/simulation/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:DetailedCircuitSimulationShape",
+      "@type": "sh:NodeShape",
+      "label": "Model simulation activity shape",
+      "comment": "Query: Get all simulation that generates a voltage trace layer 5",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "targetClass": "nsg:Simulation",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:used",
+              "name": "Model",
+              "description": "Used model.",
+              "seeAlso": "{{base}}/schemas/neurosciencegraph/simulation/modelinstance/v0.1.0/shapes/ModelInstanceShape",
+              "class": "nsg:Model",
+              "minCount": 1
+            },
+            {
+              "path": "prov:used",
+              "name": "Configuration",
+              "description": "Used configuration.",
+              "seeAlso": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape",
+              "class": "nsg:Configuration",
+              "minCount": 1
+            },
```

Comment:
In addition to the Model and the Configuration, a simulation may use input data files, e.g. containing the stimulus to be applied to the model.

Reply:
Okay. I've opened an issue (#44) so that we can further define what needs to be in the set of data used as input of a model simulation activity.